src/components/chips/Chip.vue

```diff
@@ -1,33 +1,72 @@
 <script lang="ts" setup>
+import { type StyleValue } from 'vue';
+import { logicNot, logicOr } from '@vueuse/math';
+import { objectOmit } from '@vueuse/shared';
 import { type ContextColorsType } from '@/consts/colors';
 import Icon from '@/components/icons/Icon.vue';
+import { type RuiIcons } from '~/src';
 
 export interface Props {
-  label?: string;
-  dismissible?: boolean;
+  label?: boolean;
```

Comment:
what does the label do now?

Reply:
OK so it makes it not rounded, maybe we can use a different name there to indicated that it makes it non rounded?

Reply:
I was following vuetify naming. Okay I think `tile` would be better.

Reply:
updated

Reply:
great thank you :) tile sound better yes